services/movie.py

```diff
@@ -0,0 +1,44 @@
+from django.db.models import QuerySet
+
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+
+    movies = Movie.objects.all()
+    if genres_ids is not None and actors_ids is not None:
+        return movies.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        ).distinct()
+    if genres_ids is not None and actors_ids is None:
+        return movies.filter(genres__id__in=genres_ids).distinct()
+    if genres_ids is None and actors_ids is not None:
+        return movies.filter(actors__id__in=actors_ids).distinct()

```

Comment:
You have two cases here no need to hard ifs:
1. if genres_ids is not None
2. if actors_ids is not None